internal/uvm/virtual_device.go

```diff
@@ -31,6 +36,23 @@ type VPCIDevice struct {
 	refCount uint32
 }
 
+// GetAssignedDeviceVMBUSInstanceID returns the instance ID of the VMBUS channel device node created.
+//
+// When a device is assigned to a UVM via VPCI support in HCS, a new VMBUS channel device node is
+// created in the UVM. The actual device that was assigned in is exposed as a child on this VMBUS
+// channel device node.
+//
+// A device node's instance ID is an identifier that distinguishes that device from other devices
+// on the system. The GUID of a VMBUS channel device node refers to that channel's unique
+// identifier used internally by VMBUS and can be used to determine the VMBUS channel
```

Comment:
I feel this comment is a bit confusing. I think it can be simply rewritten as `A device node's instance ID is a combination of the standard VMBUS GUID and the unique GUID assigned to the VMBUS channel device node created for this device`. Saying that `The GUID of a VMBUS channel device node refers to that channel's unique
identifier used internally by VMBUS and can be used to determine the VMBUS channel device node's instance ID` makes it sound like determining this instance ID needs some system calls or something when in reality it is a just a string concatenation operation. What do you think?